### PR TITLE
TST: updated to use pip pysat 3.0

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Updated the pysat branch used on Travis to be the official Release Candidate branch for 3.0.0.  Addresses upstream changes in pysat: https://github.com/pysat/pysat/pull/676.  May be further updated after merge of pysat PR.

## Type of change

Please delete options that are not relevant.

- Integration test

# How Has This Been Tested?

Tested locally with appropriate pysat branch

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -55,7 +54,7 @@
 
 # General information about the project.
 project = u'pysatModels'
-copyright = u'2020, Angeline G Burrell, Russell Stoneback, Jeffery Klenzing'
+cr_line = u'2020, Angeline G Burrell, Russell Stoneback, Jeffery Klenzing'
 author = u'Angeline G Burrell, Russell Stoneback, Jeffery Klenzing'
```
Comment (jklenzing) [inline docs/conf.py:58]:
```suggestion
cr_line = u'2020, Angeline G Burrell, Russell Stoneback, Jeff Klenzing'
author = u'Angeline G Burrell, Russell Stoneback, Jeff Klenzing'
```
Short form is preferred, or correct spelling

File: docs/conf.py [hunk 2]
```diff
@@ -112,64 +113,51 @@
...
+latex_elements = {}
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (master_doc, 'pysatModels.tex', u'pysatModels Documentation',
-     u'Angeline G Burrell, Russel Stoneback, and Jeffery Klenzing', 'manual'),
-]
+latex_documents = [(master_doc, 'pysatModels.tex', u'pysatModels Documentation',
```
Comment (jklenzing) [inline docs/conf.py:134]: Should we use `title = 'pysatModels Documentation'` (similar to authors) to aid in maintenance?

File: docs/conf.py [hunk 3]
```diff
@@ -112,64 +113,51 @@
...
 # -- Options for manual page output ---------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [
-    (master_doc, 'pysatModels', u'pysatModels Documentation',
-     [author], 1)
-]
-
+man_pages = [(master_doc, project, u'pysatModels Documentation',
```
Comment (jklenzing) [inline docs/conf.py:142]: A great place for a `title` variable!

File: docs/conf.py [hunk 4]
```diff
@@ -112,64 +113,51 @@
...
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
-texinfo_documents = [
-    (master_doc, 'pysatModels', u'pysatModels Documentation',
-     author, 'pysatModels', 'Tools for Model-Data analysis.',
-     'Space Physics'),
-]
-
-
+texinfo_documents = [(master_doc, project, u'pysatModels Documentation',
```
Comment (jklenzing) [inline docs/conf.py:150]: Once more, with variables!

File: docs/conf.py [hunk 5]
```diff
@@ -112,64 +113,51 @@
...
+                      author, project, 'Tools for Model-Data analysis.',
```
Comment (jklenzing) [inline docs/conf.py:151]: `description` is only used once that I found, but could be as a variable up top along with title for maintenance

File: setup.cfg [hunk 6]
```diff
@@ -1,9 +1,46 @@
...
+		   scipy
+		   xarray
+
 [coverage:run]
 
 [coverage:report]
 
 [flake8]
 max-line-length = 80
+ignore = W503
```
Comment (jklenzing) [inline setup.cfg:44]:
```suggestion
ignore = W503
exclude = docs/conf.py
```
Not sure if the exclude statement is strictly needed, as flake8 does not seem to grab the docs by default.

**Review by jklenzing — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 1 above)
Comment (jklenzing) [reply docs/conf.py:58]: Could also append the author string to cr_line to aid in maintenance

File: setup.cfg (hunk 6 above)
Comment (aburrell) [reply setup.cfg:44]: Oh, it does, but I also think it's ok to flake8 the docs.  So I would rather not exclude them.

Comment (jklenzing) [reply setup.cfg:44]: That's fine.  I think we had this originally due to some conflicts with the conf.py, but if it passes I'm happy with linting this as well.

File: docs/conf.py (hunk 1 above)
Comment (aburrell) [reply docs/conf.py:58]: Could you change to long correct spelling?  I don't trust my fingers.  Also like the idea of using authors in the cr_line.

Comment (jklenzing) [reply docs/conf.py:58]: updated

File: .github/workflows/main.yml [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+    name: Python ${{ matrix.python-version }} on ${{ matrix.os }} with numpy ${{ matrix.numpy_ver }}
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install latest dependencies
```
Comment (jklenzing) [inline .github/workflows/main.yml:31]:
```suggestion
        python-version: ${{ matrix.python-version }}

    - name: Install latest dependencies
```

**Review by jklenzing — APPROVED** [review]
Looks good.  Minor style suggestion below

**Pull request merged**